src/containers/Simularium/index.tsx

```diff
@@ -102,10 +108,23 @@ class App extends React.Component<AppProps, AppState> {
                 clearUrlParams();
                 setSimulariumController(controller);
             }
-        } else if (userTrajectoryUrl) {
+        };
+
+        const loadUserTrajectoryUrl = (
+            userTrajectoryUrl: string | string[]
+        ) => {
             const verifiedUrl = urlCheck(userTrajectoryUrl);
             const fileId = getFileIdFromUrl(verifiedUrl, parsed.id);
-            if (verifiedUrl) {
+            if (
+                verifiedUrl &&
+                fileId &&
+                USER_TRAJ_REDIRECTS.includes(verifiedUrl)
+            ) {
+                // Edge case where we want to redirect to a networked file
+                const streamingUrl = getStreamingUrl(fileId);
```

Comment:
The only improvement I would suggest is:  I was imagining a function that also hides the redirect check and hides the USER_TRAJ_REDIRECTS in the utils file.   So this code looks more like this shape:
redirect = checkForRedirect(verifiedUrl, fileId);
if (redirect) {
  history.replacestate and loadNetworkedFile
}
else if....

Reply:
Ah, ok I will try to incorporate this

Reply:
Done! Thanks for the clarification.